README.md

```diff
@@ -1,56 +1,33 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
+# Capstone/ Module one
 
+> Capstone projects are solo or group projects at the end of the each of the Microverse Core Curriculum modules. This to allow me to put into practice what have learned during during this periode at Microverse
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
```

Comment:
### **Optional**
Hi, @Jeanbulambo this review is marked optional because it was not mentioned by the previous reviewer, kindly implement the changes before you merge your code to help the recruiter understand that you actually follow the project requirement.

### **Video Presentation**

- [ ] Please recorded a video presenting the features of the project with a maximum length of 5 minutes.

- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
-  The video contains a highlight about a piece of the code the student finds interesting or is proud of making.